samples/caf_sensor_manager/remote/boards/nrf54h20pdk_nrf54h20_cpuppr.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+
+CONFIG_OPENAMP=n
```

Comment:
why is this setting needed here?

Reply:
I was wondering myself. Apparently this sample still uses openamp by default. I don't want to touch anything but nrf54h20 configuration in this PR so we will have to live with it.
@rakons - something that we should probably do - review all samples and clean them up